Code/GraphMol/Aromaticity.cpp

```diff
@@ -653,6 +656,12 @@ int setAromaticity(RWMol &mol) {
 
   VECT_INT_VECT cRings;  // holder for rings that are candidates for aromaticity
```

Comment:
note to self - these could be fixed size stack and a count, might be faster than continued memory allocation/deallocation.